kvmagent/kvmagent/plugins/ha_plugin.py

```
@@ -13,6 +13,8 @@
 from zstacklib.utils import ceph
 from zstacklib.utils import sanlock
 from zstacklib.utils import xmlobject
+from zstacklib.utils import jsonobject
```

Ruff (0.8.2)

Redefinition of unused `jsonobject` from line 4

Remove definition: `jsonobject`

(F811)